Holabb.java

```diff
@@ -0,0 +1,12 @@
+package Practicas;
+
+import java.util.Scanner;
+
+public class Holabb {
+    public static void main(String[] args){
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Hello! "+"What is your name?");
```

Comment:
This is not necessary to split them up into two string, I believe you can actually put "Hello! What is your name?"